datafusion/core/tests/sqllogictests/test_files/math.slt

```diff
@@ -87,3 +87,9 @@ query RRRRRRRR
 SELECT round(125.2345, -3), round(125.2345, -2), round(125.2345, -1), round(125.2345), round(125.2345, 0), round(125.2345, 1), round(125.2345, 2), round(125.2345, 3)
 ----
 0 100 130 125 125 125.2 125.23 125.235
+
+# atan2
+query RRRR
```

Comment:
please include also tests with nulls. Its likely gap from prev implementation
`atan2(null, 1.0), atan2(2.0, null)`

Reply:
Sure, expression tests for null cases can also be found [here](https://github.com/apache/arrow-datafusion/pull/2942/files#diff-d6e2f75f82c6d7f14f1bb07bc89293c7ef4c92c4c02241044e14ad9ea8ae213dR508), returning `NULL` seems to be expected